.circleci/config.yml

```diff
@@ -249,9 +249,10 @@ jobs:
             export EMTEST_BROWSER="xvfb-run /usr/bin/google-chrome-stable $CHROME_FLAGS_BASE $CHROME_FLAGS_WASM"
             export EMTEST_LACKS_GRAPHICS_HARDWARE=1
             export EMCC_CORES=4
-            # Just run a subset of the tests for now.  To be expanded.
-            ./emscripten/tests/runner.py browser.test_[a-f]* browser.test_pthread_* skip:browser.test_aniso
```

Comment:
can we use annotations to disable these rather than listing them here?  I think its good to have   test metatdata alongside the test itself.